### ISSUE TITLE
Name property for Combo box in 'Package manager' window is not defined

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#366]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4.Under "File" menu select "Add packages" or select 'Add package' button at the bottom of the workbook page.
5. Navigate to Combo box in 'Package manager' window and Observe Inspect property

### Expected Behavior
Name property should be properly defined for Combo box(For e.g -'Package Source' )
### Actual Behavior
Name property for Edit box is not defined and currently is " "

Narrator Behavior: Narrator is announcing as ''Xamarin.Interactive.NuGet.PackageSourceViewModel, combo box, collapsed, 0% vertically scrolled''
NVDA behavior:NVDA is announcing as 'Combo box , collapsed, Xamarin.Interactive.NuGet.PackageSourceViewModel'

### User Impact:
Users who rely on screenreader will be unable to know the Name of combo box

### Screen-Shots: 
![package_source_combobox_name_property_inspect](https://user-images.githubusercontent.com/22956130/40725271-9cca8f70-6440-11e8-93f0-dc1c1a2d131d.png)
![package_source_combobox_name_property_narrator](https://user-images.githubusercontent.com/22956130/40725272-9d0088dc-6440-11e8-8f09-e535a4e78927.png)
![package_source_combobox_name_property_nvda](https://user-images.githubusercontent.com/22956130/40725273-9d3f45cc-6440-11e8-8728-3cd5358e6fa7.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1
